tests/io/test_parquet.py

```diff
@@ -138,3 +138,21 @@ def test_parquet_roundtrip_s3(s3_resource, s3_storage_options):
     assert result.crs == df.crs
     # reading back also populates the spatial partitioning property
     assert result.spatial_partitions is not None
+
+
+def test_parquet_empty_partitions(tmp_path):
+    df = geopandas.read_file(geopandas.datasets.get_path("naturalearth_lowres"))
+    # Creating filtered dask dataframe with at least one empty partition
+    ddf = dask_geopandas.from_geopandas(df, npartitions=4)
+    ddf_filtered = ddf[ddf["pop_est"] > 1_000_000_000]
+    assert (ddf_filtered.map_partitions(len).compute() == 0).any()
+
+    basedir = tmp_path / "dataset"
+    # TODO don't write metadata file as that fails with empty partitions on
+    # inferring the schema
+    ddf_filtered.to_parquet(basedir, write_metadata_file=False)
+
+    result = dask_geopandas.read_parquet(basedir)
+    assert_geodataframe_equal(result.compute(), df[df["pop_est"] > 1_000_000_000])
+    # once one partition has no spatial extent, we don't restore the spatial partitions
```

Comment:
Why we do this? In the case of empty partitions, we can simply have `None` in the GeoSeries and it still helps a lot. Is it a safe option in case of somehow incorrectly created parquet where non-empty partitions are missing bounds? Shall we maybe offer a way around that for cases like this one?

Reply:
I understand the consequences, I just think it may be worth doing something a bit smarter here now. But let's leave it for another PR.

Reply:
Yes, especially with the current situation that we easily write empty partitions to parquet (which then will result in this kind of bboxes), this is quite annoying. 

(but for this specific case, the better solution might also be to not write empty partitions to parquet)